doc/users/explain/figures.rst

```diff
@@ -0,0 +1,188 @@
+gray region and `~.Figure.add_subplot` has added an `~.axes.Axes` artist to the
+`~.Figure` (see :ref:`figure_parts`).  A more complicated visualization can add multiple Axes to
+the Figure, colorbars, legends, annotations, and the Axes theselves can have
+multiple Artists added to them (e.g. ``ax.plot`` or ``ax.imshow``).
+
+.. _viewing_figures:
+
+Viewing Figures
+================
+
+We will discuss how to create Figures in more detail below, but first it is
+helpful to understand how to view a Figure.  This varies based on how you are
+using Matplotlib, and what :ref:`Backend <what-is-a-backend>` you are using.
+
+Notebooks and IDEs
+------------------
+
+If you are using a Notebook (e.g. `Jupyter <https://jupyter.org>`_) or an IDE
+that renders Notebooks (PyCharm, VSCode, etc), then they have a backend that
+will render the Matplotlib Figure when a code cell is executed.  One thing to
+be aware of is that the default Jupyter backend (``%matplotlib inline``) will
+by default trim or expand the figure size to have a tight box around Artists
+added to the Figure (see :ref:`saving_figures`, below).
+
+Standalone scripts and interctive use
+-------------------------------------
+
+If the user is on a client with a windowing system, there are a number of
+:ref:`Backends <what-is-a-backend>` that can be used to render the Figure to
+the screen, usually using a Python Qt, Tk, or Wx toolkit, though there is a native
+MacOS backend as well.  These are typically chosen either in the user's
+:ref:`matplotlibrc <customizing-with-matplotlibrc-files>`, or by calling
+``matplotlib.use('QtAgg')`` at the beginning of a session or script.
+
+When run from a script, or interactively (e.g. from an
+`iPython shell <https://https://ipython.readthedocs.io/en/stable/>`_) the Figure
+will not be shown until we call ``plt.show()``. The Figure will appear in
+a new GUI window, and usually will have a toolbar with Zoom, Pan, and other tools
+for interacting with the Figure.  By default, ``plt.show()`` blocks
+further interaction from the script or shell until the Figure window is closed,
+though that can be toggled off for some pusposes.  For more details, please see
+:ref:`controlling-interactive`.
+
+Note that if you are on a client that does not have access to a windowing
+system, the Figure will fallback to being drawn using the "Agg" backend, and
+cannot be viewed, though it can be :ref:`saved <saving_figures>`.
+
+.. _creating_figures:
+
+Creating Figures
+================
+
+By far the most common way to create a figure is using the
+:doc:`pyplot </tutorials/introductory/pyplot>` interface. As noted in
+:ref:`api_interfaces`, the pyplot interface serves two purposes.  One is to spin
+up the Backend and keep track of GUI windows. The other is a global state for
+Axes and Artists that allow a short-form API to plotting methods. In the
+example above, we only use pyplot for the first purpose, to create the Figure
+object, ``fig``.
```

Comment:
Continuing the discussion from https://github.com/matplotlib/matplotlib/pull/24999#discussion_r1072812820

> we only use pyplot for the first purpose, ...

That reads to me like "In the above example we are not adding the figures to the global registry".  Maybe something like:

```suggestion
Axes and Artists that allow a short-form API to plotting methods. In the
example above, we only want the first purpose, to create the Figure
object, ``fig``, but it is still added to the global registry to support the
second.
```

Reply:
OK, I understand now.  If we digress like this, I made it a bit more wordy:

```
By far the most common way to create a figure is using the
:doc:`pyplot </tutorials/introductory/pyplot>` interface. As noted in
:ref:`api_interfaces`, the pyplot interface serves two purposes.  One is to spin
up the Backend and keep track of GUI windows. The other is a global state for
Axes and Artists that allow a short-form API to plotting methods. In the
example above, we use pyplot for the first purpose, and create the Figure object,
``fig``. As a side effect ``fig`` is also added to pyplot's global state, and
can be accessed via `~.pyplot.gcf`.
```